samples/Samples.h

```diff
@@ -72,6 +72,12 @@ extern "C" {
 #define MAX_SIGNALING_CLIENT_METRICS_MESSAGE_SIZE 736 // strlen(SIGNALING_CLIENT_METRICS_JSON_TEMPLATE) + 20 * 10
 #define MAX_ICE_AGENT_METRICS_MESSAGE_SIZE        113 // strlen(ICE_AGENT_METRICS_JSON_TEMPLATE) + 20 * 2
 
+#define ADJUSTMENT_INTERVAL_SECONDS 1 * HUNDREDS_OF_NANOS_IN_A_SECOND
+#define MIN_VIDEO_BITRATE_KBPS      512     // Unit kilobits/sec. Value could change based on codec.
+#define MAX_VIDEO_BITRATE_KBPS      2048000 // Unit kilobits/sec. Value could change based on codec.
```

Comment:
Are these values from a sample video?

Reply:
Its for x264enc. Does not matter what the source is. I am setting min to 512kbps since it is the value we set in our samples. As such gstreamer min for x264enc is 100kbps if I not wrong.